lib/features/home_view/widgets/departments_section/department_section.dart

```diff
@@ -24,9 +25,7 @@ class DepartmentSection extends ConsumerWidget {
           title: context.localize.departments,
           actionTitle: context.localize.list,
           onClick: () {
-            ref
-                .read(bottomNavBarControllerProvider.notifier)
-                .goTo(NavBarEnum.faculties);
+            ref.read(navigatorProvider).changeTabBar(NavBarEnum.faculties);
```

Comment:
Maybe we can extract this to a separate variable just like in the `buildings_section.dart` in sake of consistence. 

 ```dart
 static void goToFacultiesTab(WidgetRef ref) =>
      ref.read(navigatorProvider).changeTabBar(NavBarEnum.mapp);
```

Reply:
sure